spec/controllers/api/prosecution_cases_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::ProsecutionCasesController, type: :controller do
```

Comment:
**not a blocker**. I avoid controller specs generally and i see you have request specs that [cover this here](https://github.com/ministryofjustice/laa-court-data-adaptor/pull/61/files#diff-e01c816e3c08acaf4883309f914d0904)

Reply:
The controller specs are only validating the response status codes, while the request specs do a bit thorough check of the schema. I would want to keep these in to easily ensure that we have non-happy status codes tested